docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,122 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+```
+
+Create the service account using the `kubectl` command tool:
+
+```bash
+kubectl create -f rbac-tiller.yaml
+```
+
+Next, we use the `helm` client to install the Tiller server to the cluster:
+
+```bash
+helm init --service-account tiller
+```
+
+Voila, you have installed Helm to your cluster.
+
+## Step 3: Install Kubeapps
+
+Kubeapps is a open-source web-based UI for deploying and managing applications in Kubernetes clusters. It provides you a dashboard to:
+
+    - Browse Helm charts from public or your own private chart repositories and deploy them to your cluster
+    - Upgrade, manage and delete the applications that are deployed in your cluster
+    - Browse and provision external services from the Service Catalog
+
+Kubeapps can be installed using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps) from the official [Bitnami charts repository](https://github.com/bitnami/charts).
+
+Begin by configuring the Helm client to use the Bitnami charts repository.
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+Now you can easily install Kubeapps to your Kubernetes cluster with the following command:
+
+```bash
+helm install --name kubeapps --namespace kubeapps bitnami/kubeapps \
+    --set mongodb.metrics.enabled=true \
+    --set ingress.enabled=true \
+    --set ingress.hosts[0].name=kubeapps.[YOUR-BKPR-ZONE] \
+    --set ingress.hosts[0].tls=true \
+    --set ingress.hosts[0].tlsSecret=kubeapps-tls \
+    --set ingress.hosts[0].certManager=true
+```
+
+The command line flags provided in the above command enable the Ingress resource in the Kubeapps chart and also enable TLS support. Once installed, you will be able to access the Kubeapps dashboard securely (HTTPS) over the internet at `https://kubeapps.[YOUR-BKPR-ZONE]`.
+
+_Please replace the placeholder string `[YOUR-BKPR-ZONE]` in the above command with the DNS zone configured while setting up BKPR in you Kubernetes cluster._
+
+However, if you not would like the dashboard to be accessible externally over the Internet, please disable the Ingress support while installing the chart and follow the post-install instructions to access the dashboard using a proxy connection to the cluster.
+
+The chart parameter `mongodb.metrics.enabled=true` in the above command, enables the Prometheus exporter for MongoDB in the Kubeapps chart.
+
+> **Tip**:
+>
+> When installing applications to your cluster, remember to enable Prometheus exporters in applications that support them. The exported instrumentation data will be automatically scrapped by the Prometheus server installed in the cluster by BKPR and will provide useful insights into the clusters performance.
```

Comment:
cluster's performance

install to -> install on

Should we link to our Prometheus component doc here for more info?